distributed/shuffle/_worker_extension.py

```diff
@@ -695,11 +702,10 @@ async def _get_shuffle_run(
             shuffle = await self._refresh_shuffle(
                 shuffle_id=shuffle_id,
             )
-        if run_id < shuffle.run_id:
-            raise RuntimeError("Stale shuffle")
-        elif run_id > shuffle.run_id:
-            # This should never happen
-            raise RuntimeError("Invalid shuffle state")
+        if run_id > shuffle.run_id:
```

Comment:
Minor: flip order of operands to align with `restrict_task` in the scheduler extension?

Reply:
Good point, done.